screen-capture/getdisplaymedia.https.html

```diff
@@ -18,9 +17,7 @@
 const j = obj => JSON.stringify(obj);
 
 async function getDisplayMedia(constraints) {
-  const p = new Promise(r => button.onclick = r);
-  await test_driver.click(button);
-  await p;
+  await test_driver.bless("getDisplayMedia");
```

Comment:
I'd suggest making the call synchronously in the `action` callback, so that we're definitely still blessed by the click. (Avoiding any browser differences in timing here.)

```suggestion
  await test_driver.bless("getDisplayMedia", () => navigator.mediaDevices.getDisplayMedia(constraints));
```